packages/tabs/src/Tabs.ts

```diff
@@ -386,8 +403,10 @@ export class Tabs extends SizedMixin(Focusable) {
     }
 
     private updateCheckedState = (): void => {
-        if (!this.tabs.length) {
-            this.tabs = [...this.querySelectorAll('[role="tab"]')] as Tab[];
+        if (!this.tabs.length && this.slotEl) {
+            this.tabs = this.slotEl
+                .assignedElements()
+                .filter((el) => el.getAttribute('role') === 'tab') as Tab[];
```

Comment:
we could consider pulling this query into a function in both places it's used:
```suggestion
            this.tabs = this.assignedTabs()
```